pkg/cli/cmds/root.go

```diff
@@ -124,6 +125,16 @@ func validateCISreqs() error {
 	return nil
 }
 
+// setCISFlags validates and sets any CLI flags necessary to ensure
+// compliance with the profile.
+func setCISFlags(clx *cli.Context) error {
+	// If the user has specifically set this to false, raise an error
+	if clx.IsSet(pkdFlagName) && !clx.Bool(pkdFlagName) {
+		return fmt.Errorf("--%s must be true when using --profile=%s", pkgFlagName, profile)
```

Comment:
```suggestion
		return fmt.Errorf("--%s must be true when using --profile=%s", pkdFlagName, profile)
```

Reply:
thanks... not sure why my editor didn't catch that one.